python/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+
+IF(NOT Kokkos_ENABLE_PYBIND11)
+    return()
```

Comment:
I'd rather skip the `add_subdirectory`?

Reply:
I like this technique because it enables the CMakeLists.txt to show up in my IDE but your call